targets/ChibiOS/_nanoCLR/nanoFramework.Hardware.Rpi/nf_hardware_rpi_native_nanoFramework_Hardware_Rpi_Pio_PioStateMachine.cpp

```
@@ -0,0 +1,401 @@
+    g_PioClaimedSm[param0] &= ~(1u << sm);
+}
+
+void PioStateMachine::NativeSetConsecutivePinDirs(
+    signed int param0,
+    signed int param1,
+    signed int param2,
+    signed int param3,
+    bool param4,
+    HRESULT &hr)
+{
+    PIO_TypeDef *pio = PioFromIndex(param0);
```

Clang (14.0.6)

[warning] variable 'pio' is not initialized

(cppcoreguidelines-init-variables)